### PR TITLE
Use go-to-protobuf from gengo

### PR DESCRIPTION
**Pull request by lavalamp**
Missed this in the previous change.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33412)

### CODE REVIEW

File: vendor/k8s.io/gengo/examples/go-to-protobuf/protobuf/cmd.go [hunk 1]
```diff
@@ -78,7 +78,6 @@ func New() *Generator {
 			`k8s.io/kubernetes/federation/apis/federation/v1beta1`,
 			`k8s.io/kubernetes/pkg/apis/certificates/v1alpha1`,
 			`k8s.io/kubernetes/pkg/apis/imagepolicy/v1alpha1`,
-			`k8s.io/kubernetes/pkg/apis/storage/v1beta1`,
```
Comment (lavalamp) [inline vendor/k8s.io/gengo/examples/go-to-protobuf/protobuf/cmd.go]: Uh, I didn't notice this until I made this PR. @smarterclayton do these defaults really matter?

**Comment by smarterclayton** [comment]
They should be replicable from the CLI - openshift uses a different set and
is able to run successfully.

**Comment by lavalamp** [comment]
Cool. We should delete the defaults, then. I'll do it in a followup.

**Comment by smarterclayton** [comment]
I deleted the defaults in the defaulter PR I opened and suffered no
consequences (main.go still in Kube).

File: vendor/k8s.io/gengo/examples/go-to-protobuf/protobuf/cmd.go (hunk 1 above)
Comment (wojtek-t) [inline vendor/k8s.io/gengo/examples/go-to-protobuf/protobuf/cmd.go]: They do - it basically defines what we are generating.

Comment (smarterclayton) [inline vendor/k8s.io/gengo/examples/go-to-protobuf/protobuf/cmd.go]: This was never part of Tim's changes to make

**Comment by smarterclayton** [comment]
@ncdc you changed this code recently right?

On Oct 27, 2016, at 9:41 AM, Kubernetes Submit Queue <
notifications@github.com> wrote:

This PR hasn't been active in 30 days. It will be closed in 59 days (Dec
26, 2016).

cc @lavalamp <https://github.com/lavalamp> @smarterclayton
<https://github.com/smarterclayton>

You can add 'keep-open' label to prevent this from happening, or add a
comment to keep it open another 90 days

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/33412#issuecomment-256643607>,
or mute the thread

**Comment by ncdc** [comment]
@smarterclayton yeah in #35432. I can make the corresponding change in gengo if needed?

**Comment by smarterclayton** [comment]
Please, I forgot to ask.

**Comment by ncdc** [comment]
Upstream gengo PR for dash-to-underscore conversion: https://github.com/kubernetes/gengo/pull/16

**Pull request closed without merging**